composer.json

```diff
@@ -27,11 +27,11 @@
         "phpunit/phpcov": "~3.0|~4.0",
         "phpunit/phpunit": "~5.7.14|~6.1|~7.0",
         "laravel/framework": "~5.4.36|~5.5.0|~5.6.0",
-        "orchestra/testbench": "~3.4.0|~3.5.0|~3.6.0",
+        "orchestra/testbench": "~3.4.11|~3.5.0|~3.6.0",
```

Comment:
`~` is effectively a wildcard selector, so `~3.4.0` already includes `~3.4.11`.  This change is unnecessary (same for the 2 testbench entries below this.

Reply:
If you call `composer install --prefer-dist` it will install the lowest version of `orchestra/testbench` = `3.4.0`, but this version isn't compatible. To fix this, we need to set the minimum to `3.4.11`. Otherwise TravisCI will fail.